pkg/receiver/smartagent/README.md

```diff
@@ -2,50 +2,49 @@
+[Smart Agent release bundle](https://github.com/signalfx/signalfx-agent/releases/latest),
+which is installed by the Splunk Distribution of OpenTelemetry Collector.
 
-**Beta: All Smart Agent monitors are supported at this time.  Configuration and behavior may change without notice.**
+While this document outlines some of the capabilities of the Smart Agent receiver, please see the
+[migration guide](../../../docs/signalfx-smart-agent-migration.md)
+for more information about migrating from the Smart Agent to the Splunk Distribution of the OpenTelemetry Collector.
+
+**Beta: All Smart Agent monitors are supported at this time. Configuration and behavior may change without notice.**
 
 ## Configuration
 
-Each `smartagent` receiver configuration acts as a drop-in replacement for each supported Smart Agent Monitor
-[configuration](https://github.com/signalfx/signalfx-agent/blob/main/docs/monitor-config.md) with some exceptions:
+For each Smart Agent 
+[monitor](https://github.com/signalfx/signalfx-agent/blob/main/docs/monitor-config.md)
+you want to put into the Collector, add a `smartagent` receiver configuration block. Once configured in the Collector, each
+`smartagent` receiver will act as a drop-in replacement for its corresponding Smart Agent monitor.
 
-1. Any Agent global or collectd desired configuration should be performed via the
-[Smart Agent Extension](../../extension/smartagentextension/README.md).
-1. In lieu of `discoveryRule` support, the Collector's
-[`receivercreator`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/receivercreator/README.md)
-and associated [Observer extensions](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/observer/README.md)
-should be used.
-1. The [`signalfx-forwarder`](https://github.com/signalfx/signalfx-agent/blob/main/docs/monitors/signalfx-forwarder.md)
-monitor should be made part of both `metrics` and `traces` pipelines utilizing the
-[`signalfx`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/signalfxexporter/README.md)
-and [`sapm`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/sapmexporter/README.md) exporters, respectively.
-1. All metric content replacement and transformation rules should utilize existing
+1. Put any Smart Agent or collectd configuration into the
+[Smart Agent Extension section](../../extension/smartagentextension/README.md)
+of your Collector configuration.
+1. Instead of using Smart Agent's `discoveryRule`s, use the Collector's
+[Receiver Creator](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/receivercreator/README.md)
+and [Observer extensions](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/observer/README.md).
+1. Instead of using the [SignalFx Forwarder](https://github.com/signalfx/signalfx-agent/blob/main/docs/monitors/signalfx-forwarder.md)
```

Comment:
The signalfx-forwarder monitor in both metrics and trace pipelines was the important bit and it isn't replaced by the   sapm exporter.

Reply:
Nice catch. Thanks.